controller/model/edge_router_handlers.go

```diff
@@ -305,9 +307,6 @@ func (handler *EdgeRouterHandler) ReEnroll(router *EdgeRouter) error {
 		return err
 	}
 
-	log.Info("clearing cached fingerprint for re-enrolling edge router")
-	handler.env.GetHostController().GetNetwork().Routers.UpdateCachedFingerprint(router.Id, "")
```

Comment:
Can you point me to or briefly explain the change in the fabric level that negates this need?

Reply:
https://github.com/openziti/fabric/pull/320/files#diff-2f9de7e5ec0b4ac907c5a3414a4a0097b269b3cd372d6a5e300d41d8227e65b6R153
and
https://github.com/openziti/fabric/pull/320/files#diff-2f9de7e5ec0b4ac907c5a3414a4a0097b269b3cd372d6a5e300d41d8227e65b6R294

Reply:
Cool.